docs/source/developer/extension_tutorial.rst

```diff
@@ -63,7 +63,7 @@ new environment named ``jupyterlab-ext``.
 
 .. code:: bash
 
-    conda create -n jupyterlab-ext -c conda-forge --override-channels jupyterlab cookiecutter nodejs git
+    conda create -n jupyterlab-ext -c conda-forge -c defaults --override-channels jupyterlab cookiecutter nodejs git
```

Comment:
This won't work if you've redefined the `defaults` channel. Can you do this instead?

```
conda create -n jupyterlab-ext --override-channels --strict-channel-priority -c conda-forge -c anaconda jupyterlab cookiecutter nodejs git
`--override-channels --strict-channel-priority -c conda-forge -c anaconda`

Reply:
Thanks for the feedback.